doc/conf.py

```diff
@@ -54,6 +54,8 @@
 
 autosummary_generate = True
 
+autodoc_typehints = "none"
```

Comment:
it seems this did the trick, no?

Reply:
Yup just removing them so there's nothing to resolve. I'm confused because it works locally and the link will correctly take me to numpy's API. It seems to be a bug of autodoc related to https://github.com/sphinx-doc/sphinx/issues/10785. I'll document this in the issue I opened (#4018) and we can discuss how to make the docs for typed classes and functions look better.

Reply:
this sounds like a wonderful omen when we'll start adding type annotations to the code base 😨